### DISCUSSION TITLE
plugin for wikipedia references

### DISCUSSION THREAD

**Discussion by ZehuaZhang** [remarkjs/remark#1192] [Q&A] [answered]
The syntax for wikipedia reference is `^[[name]](url)^`, is there any plugin to support that, or how I could make it work using component override with regex matching the pattern? Since I am here, is there any general documentation for developing such enhancements?

**Comment by ChristianMurphy** [comment, marked as answer]
There are several plugins for wiki links.
The state of repair and closeness to wikipedia's semi standard format may vary.

* https://github.com/landakram/remark-wiki-link
* https://github.com/C1200/remark-wikilinks
* https://github.com/wikibonsai/remark-wikirefs

> Since I am here, is there any general documentation for developing such enhancements?

https://github.com/micromark/micromark#extending-markdown and https://github.com/micromark/micromark#creating-a-micromark-extension

**Reply by ZehuaZhang** [reply]
Much appreciated!
